packages/react-router-dom/package.json

```diff
@@ -45,12 +45,12 @@
   },
```

Comment:
We still publish this to npm and not artifactory, right?

Reply:
We can change that practice. We should stop publishing this to npm